tests/integration/ovs_test.py

```diff
@@ -31,6 +31,8 @@
 
 from .testlib import assertlib
 from .testlib import cmdlib
+from .testlib.nmlib import get_ovs_port_by_slave
+from .testlib.nmlib import list_profiles_by_iface_name
```

Comment:
I think it would be better to do `from .testlib import nmlib`. Not a must, just a nit. :-)

Reply:
Done